lib/pure/memfiles.nim

```diff
@@ -245,3 +245,35 @@ proc close*(f: var MemFile) =
   
   if error: raiseOSError(lastErr)
 
+type Record* {.unchecked.} = object
```

Comment:
Lower case members here please. I would also prefer some documentation as to what the record type's purpose is. Should it be mutable? Immutable? (Really, I would prefer that the stdlib just get an immutable string type, then that could be used)